apps/web/app/components/radio-card.jsx

```diff
@@ -18,91 +18,128 @@ const RadioCard = ({
   url,
   // eslint-disable-next-line react/prop-types
   country,
+  // eslint-disable-next-line react/prop-types
 }) => {
   const { player, setPlayer } = usePlayer();
+
   return (
+    //Main
     <div
-      key={stationuuid}
-      className={`bg-white rounded-lg shadow-md p-4 ${
-        player.stationId === stationuuid ? "animate-pulse" : ""
-      }`}
+      className={`flex flex-col flex-wrap max-w-sm mx-auto bg-white rounded-xl border border-gray-200 overflow-hidden p-4 h-[220px] w-[268px] flex-shrink-0 justify-between`}
     >
-      <button
-        onClick={() =>
-          setPlayer(
-            stationuuid == player.stationId
-              ? {}
-              : { name, url, stationId: stationuuid, country },
-          )
-        }
-        className="play-btn bg-blue-500 text-white rounded-full p-2 mb-2 hover:bg-blue-600 transition duration-300"
-      >
-        {player.stationId === stationuuid ? (
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            className="h-6 w-6 stop-icon"
-            fill="none"
-            viewBox="0 0 24 24"
-            stroke="currentColor"
+      {/* Title,likes, count */}
+      <div className={`flex justify-between`}>
+        <div
+          className={`flex items-center flex-shrink-0 h-11 w-11 bg-gradient-to-tr from-[#5539B2] to-[#D4C7FD] rounded-full flex items-center justify-center text-white text-xs font-semibold select-none`}
+        >
+          {name.slice(0, 2).toUpperCase()}
+        </div>
+        <div className={`flex flex-col justify-around pl-2`}>
+          <div className={`text-base font-semibold text-gray-900`}>
+            <Truncate>{name}</Truncate>
+          </div>
+          <div className={`text-xs text-[#00192CA3]/[0.64]`}>
+            {clickcount} kişi dinliyor • {votes} beğeni
+          </div>
+        </div>
+      </div>
+
+      {/* Tag */}
+      <div className="h-[60px] ml-0.5 flex flex-wrap gap-1.5 select-none justify-start">
+        {tags &&
+          tags
+            .split(",")
+            .slice(0, 4)
+            .map((tag, index) => (
+              <button
+                key={index}
+                className="h-[27px] px-2 py-1 bg-blue-100 text-blue-800 rounded-lg font-normal text-xs"
+              >
+                {tag.trim().charAt(0).toUpperCase() + tag.trim().slice(1)}
```

Comment:
Sanırım ilk harifini büyük yapmak için bunu yaptın ama bunu CSS ile daha kolay yapabilirsin. Tailwind'de vardır.

Reply:
Yeni commit'e çözdüm sanıyorum. Yorumlarınızı beklerim..